spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/ZuulProperties.java

```diff
@@ -244,8 +244,11 @@ public void init() {
 
 		private boolean customSensitiveHeaders = false;
 
+		private Set<String> methods = new LinkedHashSet<>();
```

Comment:
I prefer a way to do this, so that this class doesn't change for every new way of matching.